node/test/Test/Pos/Update/PollSpec.hs

```diff
@@ -152,3 +162,64 @@ applyActions ps actionList =
         resultPStates = ps : scanl Poll.execPurePollWithLogger ps pollSts
         newPollStates = scanl (flip Poll.modifyPollState) ps resultModifiers
     in conjoin $ zipWith (===) resultPStates newPollStates
+
+emptyPollSt :: (BlockVersion, BlockVersionData) -> Poll.PollState
+emptyPollSt bvInfo = Poll.PollState
+    mempty
+    bvInfo
+    mempty
+    mempty
+    mempty
+    mempty
+    mempty
+    mempty
+    mempty
+    mempty
+
+perform :: [PollAction] -> Poll.PurePoll ()
+perform = foldl (>>) (return ()) . map actionToMonad
```

Comment:
Isn't `foldl (>>) (return ())` the same as `sequence`?

Reply:
http://hackage.haskell.org/package/base-4.10.0.0/docs/src/Data.Foldable.html#sequence_

```
sequence_ :: (Foldable t, Monad m) => t (m a) -> m ()
sequence_ = foldr (>>) (return ())
```

I need it to be a `foldl` here. I can just reverse the list, but I'd rather not overengineer this.